include/silo/storage/column/float_column.h

```diff
@@ -1,31 +1,57 @@
 #ifndef SILO_FLOAT_COLUMN_H
 #define SILO_FLOAT_COLUMN_H
 
+#include <deque>
 #include <string>
 #include <vector>
 
-#include "silo/storage/column/column.h"
+namespace boost::serialization {
+struct access;
+}
+
+namespace silo {
+struct Database;
```

Comment:
Why do we need this here? This is again a cyclic dependency: database -> columns -> float column -> database.